econml/ortho_forest.py

```diff
@@ -807,10 +857,13 @@ def parameter_estimator_func(Y, T, X,
             diagonal[0] = 0
             reg = lambda_reg * np.diag(diagonal)
             # Ridge regression estimate
-            param_estimate = np.linalg.lstsq(np.matmul(weighted_X_aug.T, X_aug) + reg,
-                                             np.matmul(weighted_X_aug.T, pointwise_params), rcond=None)[0].flatten()
-            # Parameter returned by LinearRegression is (d_T, )
-            return param_estimate
+            linear_coef_estimate = np.linalg.lstsq(np.matmul(weighted_X_aug.T, X_aug) + reg,
+                                                   np.matmul(weighted_X_aug.T, pointwise_params),
+                                                   rcond=None)[0].flatten()
+            X_aug = np.append([1], X_single)
```

Comment:
Same question about case when X_single is None.

Reply:
Fixed! See other comment.